table_to_graph_importer/table_to_graph_importer.ipynb

```diff
@@ -0,0 +1,393 @@
+    "        },\n",
+    "        'name_mappings': {\n",
+    "            'individual': {'label': 'INDIVIDUAL'},\n",
+    "            'address': {'label': 'ADDRESS'}\n",
+    "        },\n",
+    "        'one_to_many_relations': {\n",
+    "            'address': [], \n",
+    "            'individual': [{\n",
+    "                'foreign_key': {\n",
+    "                    'column_name': 'add_id',\n",
+    "                    'reference_table': 'address',\n",
+    "                    'reference_key': 'add_id'\n",
+    "                },\n",
+    "                'label': 'LIVES_IN'\n",
+    "            }]\n",
+    "        }\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "947c2325",
+   "metadata": {},
+   "source": [
+    "## 6. Importing from local file system\n",
+    "\n",
+    "Finally, to import the data into Memgraph, all you need to do is make an instance of `LocalFileSystemImporter` and call its `translate()` method."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08161204",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from gqlalchemy.loaders import LocalFileSystemImporter\n",
+    "\n",
+    "importer = LocalFileSystemImporter(\n",
+    "    file_extension=\"csv\",\n",
+    "    data_configuration=data_conf,\n",
+    "    local_storage_path=\"./\",\n",
+    ")\n",
+    "\n",
+    "importer.translate(drop_database_on_start=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d9da1f4",
+   "metadata": {},
+   "source": [
+    "## 7. Importing from online file storage\n",
+    "\n",
+    "Furthermore, we offer support for reading data from online storage providers **Amazon S3** and **Azure Blob** via `AmazonS3Importer` and `AzureBlobImporter`. To use the Azure storage, for example, simply use the following snippet:\n",
+    "\n",
+    "```python\n",
+    "importer = AzureBlobImporter(\n",
+    "    file_extension=\"parquet\",\n",
+    "    data_configuration=data_conf,\n",
+    "    blob_account_name=\"<your_account_name>\",\n",
+    "    blob_account_key=\"<your_account_key>\",\n",
+    "    container_name=\"test\"\n",
+    ")\n",
+    "```\n",
+    "\n",
+    "For any other service provider, it is possible to implement your custom importer class, [here's how](link-to-how-to-guide). Don't forget gqlalchemy is open source, so you can submit your extended functionality on our [github](https://github.com/memgraph/gqlalchemy)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d46b4c05",
+   "metadata": {},
+   "source": [
+    "## 8. Querying the database and retrieving results\n",
+    "\n",
+    "Let's make sure that our data was imported correctly by retrieving it:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7da3d5a7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = memgraph.execute_and_fetch(\n",
+    "    \"\"\"\n",
+    "    \"\"\"\n",
+    ")"
```

Comment:
this one is empty, did you mean to pass here a query or you just put this by accident?

Reply:
yes, this is in TODO state, will complete